package.json

```diff
@@ -62,9 +62,10 @@
     "karma-firefox-launcher": "^1.1.0",
     "karma-jasmine": "^2.0.1",
     "karma-jasmine-html-reporter": "^1.4.0",
-    "protractor": "^5.4.2",
-    "ts-node": "^7.0.1",
+    "protractor": "~5.4.2",
+    "redux-devtools-extension": "^2.13.7",
```

Comment:
The `redux-devtools-extension` dependency is not necessary and we should remove it in the `package.json` file.

Reply:
Removed it. Sorry for responding late, I noticed this just now.